server/container_create.go

```diff
@@ -597,6 +578,9 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	}
 
 	containerImageConfig := containerInfo.Config
+	if containerImageConfig == nil {
+		return nil, fmt.Errorf("empty image config for %s", image)
+	}
```

Comment:
@sameo this is the first place we get the config. It makes sense to me to have this check here, otherwise code like at line 601/585 would panic before even reaching the processArgs function.